airbyte-oauth/src/main/java/io/airbyte/oauth/flows/google/YouTubeAnalyticsOAuthFlow.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.oauth.flows.google;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import io.airbyte.config.persistence.ConfigRepository;
+import java.net.http.HttpClient;
+import java.util.function.Supplier;
+
+public class YouTubeAnalyticsOAuthFlow extends GoogleOAuthFlow {
+
+  public static final String SCOPE_URL = "https://www.googleapis.com/auth/yt-analytics.readonly";
```

Comment:
I believe this constant could be `private`. Do we use it outside the class?

Reply:
I am no sure, I am not java expert :)
I just copy-paste from this 
`./airbyte-oauth/src/main/java/io/airbyte/oauth/flows/google/GoogleAnalyticsOAuthFlow.java`

Reply:
fixed